spec/models/miq_group_spec.rb

```diff
@@ -270,14 +270,13 @@
     it "adds new groups after initial seed" do
       [Tenant, MiqUserRole, MiqGroup].each(&:seed)
 
-      role_map_path = File.expand_path(File.join(Rails.root, "db/fixtures/role_map.yaml"))
+      role_map_path = Rails.root.join("db/fixtures/role_map.yaml")
       role_map = YAML.load_file(role_map_path)
-      role_map.unshift('EvmRole-test_role' => 'tenant_quota_administrator')
-      filter_map_path = File.expand_path(File.join(Rails.root, "db/fixtures/filter_map.yaml"))
-      filter_map = YAML.load_file(filter_map_path)
+      role_map = {'EvmRole-test_role' => 'tenant_quota_administrator'}.merge(role_map)
+      filter_map_path = Rails.root.join("db/fixtures/filter_map.yaml")
 
       allow(YAML).to receive(:load_file).with(role_map_path).and_return(role_map)
-      allow(YAML).to receive(:load_file).with(filter_map_path).and_return(filter_map)
+      allow(YAML).to receive(:load_file).with(filter_map_path).and_call_original
```

Comment:
Sadly, can't do that as RSpec won't match on a Pathname object like that. Expects String.